Packs/Active_Directory_Query/Integrations/Active_Directory_Query/README.md

```diff
@@ -355,6 +386,10 @@ Creates an Active Directory contact.
 #### Base Command
 
 `ad-create-contact`
+
+##### Required Permissions
+Requires `Create, delete, and manage user accounts` permissions.
```

Comment:
R u sure there is no way permission for only for contacts (and not for user accounts)?
![image](https://github.com/demisto/content/assets/88268646/cbf52506-3ec1-4ffa-a102-44a43ee22290)

Reply:
I dint find anything about this specific permission but `Create, delete, and manage user accounts` should be enough

Reply:
I don't like the option for delete here - should be only READ / Write all properties.
And only for contact objects.

<img width="565" alt="image" src="https://github.com/demisto/content/assets/88268646/6c28d8b2-bb90-42ab-8e0a-56c9076460ae">
<img width="528" alt="image" src="https://github.com/demisto/content/assets/88268646/ed8ead94-da5a-4f3c-8436-e365aa884d55">